docs/panels.rst

```diff
@@ -189,6 +189,17 @@ line_profiler_.
 
 .. _line_profiler: https://github.com/rkern/line_profiler
 
+LDAP Tracing
```

Comment:
Can you move this entire section above `Line Profiler` so it's alphabetical?

Reply:
Fixed that! Thank you